torchgeo/datasets/urban3d.py

```diff
@@ -0,0 +1,281 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Urban3DChallenge dataset."""
```

Comment:
```suggestion
"""Urban 3D Challenge dataset."""
```
We should use the dataset name, not the class name